components/modelregistry/modelregistry.go

```diff
@@ -124,3 +137,28 @@ func (m *ModelRegistry) ReconcileComponent(ctx context.Context, cli client.Clien
 	}
 	return nil
 }
+
+//go:embed resources/servicemesh-member.tmpl.yaml
+var smmTemplate string
+
+func createServicemeshMember(ctx context.Context, cli client.Client, dscispec *dsciv1.DSCInitializationSpec, namespace *corev1.Namespace) error {
```

Comment:
nit: `createServiceMeshMember`, but perhaps better to name it `enrollToServiceMesh` so it represents the intent, not the current implementation of it?

Reply:
Renamed